cypress/e2e/marc/marc-authority/reporting-marc-authority/marc-autority-reporting-updated-heading.cy.js

```diff
@@ -0,0 +1,213 @@
+    () => {
+      MarcAuthorities.searchBy(testData.searchOption, testData.title);
+      MarcAuthoritiesSearch.selectAuthorityByIndex(0);
+      MarcAuthority.edit();
+      QuickMarcEditor.waitLoading();
+
+      // Waiter needed for the whole page to be loaded.
+      cy.wait(2000);
+      QuickMarcEditor.updateExistingField(testData.tag100, testData.updatedTag100Value);
+      QuickMarcEditor.saveAndCloseUpdatedLinkedBibField();
+      QuickMarcEditor.confirmUpdateLinkedBibsKeepEditing(1);
+
+      MarcAuthorities.clickActionsAndReportsButtons();
+      MarcAuthorities.fillReportModal(today, tomorrow);
+      MarcAuthorities.clickExportButton();
+
+      cy.intercept('POST', '/data-export-spring/jobs').as('getId');
+      cy.wait('@getId', { timeout: 10000 }).then((item) => {
+        const jobID = item.response.body.name;
+        const expectedJobData = [
+          jobID,
+          expectedJobValues.status,
+          expectedJobValues.jobType,
+          expectedJobValues.description,
+          testData.userProperties.username,
+          todayWithoutPaddingZero,
+        ];
+        const expectedJobDetails = {
+          jobID,
+          status: expectedJobValues.status,
+          jobType: expectedJobValues.jobType,
+          outputType: expectedJobValues.outputType,
+          description: expectedJobValues.description,
+          source: testData.userProperties.username,
+          startDate: todayWithoutPaddingZero,
+          endDate: todayWithoutPaddingZero,
+        };
+        MarcAuthorities.checkCalloutAfterExport(jobID);
+        cy.visit(TopMenu.exportManagerPath);
+        ExportManagerSearchPane.waitLoading();
+        ExportManagerSearchPane.searchByAuthorityControl();
+        ExportManagerSearchPane.verifyJobDataInResults(expectedJobData);
+        ExportManagerSearchPane.verifyResultAndClick(jobID);
+        ExportManagerSearchPane.verifyJobDataInDetailView(expectedJobDetails);
+        ExportManagerSearchPane.downloadLastCreatedJob(item.response.body.name);
+      });
+      const downloadedReportDate = DateTools.getFormattedDate({ date: new Date() });
+      const fileNameMask = `${downloadedReportDate}*`;
+      FileManager.verifyFile(
+        MarcAuthorities.verifyMARCAuthorityFileName,
+        fileNameMask,
+        MarcAuthorities.verifyContentOfExportFile,
+        [
+          'Last updated',
+          `${downloadedReportDate}`,
+          'Original heading',
+          testData.authorityHeading,
+          'New heading',
+          testData.authorityHeading,
+          'Identifier',
+          testData.authority001FieldValue,
+          'Original 1XX',
+          '100',
+          'New 1XX',
+          '100',
+          'Authority source file name',
+          'LC Name Authority file (LCNAF)',
+          'Number of bibliographic records linked',
+          '1',
+          'Updater',
+        ],
+      );
+      FileManager.deleteFolder(Cypress.config('downloadsFolder'));
```

Comment:
I would suggest not to delete entire folder as this could affect other tests running in parallel. Please remove single file